odoo/addons/base/static/src/js/res_config_settings.js

```diff
@@ -187,8 +187,7 @@ var BaseSettingRenderer = FormRenderer.extend({
         this.searchInput.val("");
         _.each(this.modules, function (module) {
             module.settingView.addClass('o_hidden');
-            module.settingView.find('.o_setting_box').removeClass('o_hidden');
-            module.settingView.find('h2').removeClass('o_hidden');
+            module.settingView.find('h2, .o_setting_box, .o_setting_tip').removeClass('o_hidden');
```

Comment:
maybe you can say in your commit message that we don't want "setting box css" for those elements (it's 1 more reason to add this new class)